src/modules/main/controllers/admin-read.controller.ts

```diff
@@ -260,6 +268,14 @@ export class AdminReadController {
     return mapper.createCollectionDetailDto(item);
   }
 
+  @Get('collection/trial/:id')
+  async getCollectionEvaDetail(@Param('id', ParseUUIDPipe) id: CollectionId) {

```

Comment:
we already have endpoints to return collections, moreover this endpoint is by no means bound to current user